virtuaaliviivakoodi/deconstructors/date.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from typing import Union
+
+
+def deconstruct_date(virtuaaliviivakoodi: str) -> Union[datetime.date, None]:
+    """Deconstructs the due date from virtuaaliviivakoodi from "yymmdd" to datetime object.
+
+    The date can be not defined, in which case it is "000000" in the virtuaaliviivakoodi.
+    In this case None is returned.
+    """
+
+    due_date = virtuaaliviivakoodi[48:]
```

Comment:
Maybe use exact date boundaries?

```suggestion
    due_date = virtuaaliviivakoodi[48:54]
```

Reply:
Also, I'd probably save boundaries as a constant e.g.:
(double check the slices, I did not do that 😄)

```py
class VirtuaaliviivakoodiSlice:
    # fmt: off
    SYMBOL             = slice(0, 1)
    IBAN               = slice(1, 17)
    REFERENCE_FIN      = slice(28, 48)
    REFERENCE_RF_HEAD  = slice(25, 27)
    REFERENCE_RF_TAIL  = slice(28, 48)
    DATE               = slice(48, 54)
    # fmt: on
```

Then, different parts of virtuaaliviivakoodi can be accessed as follows:

```py
virtuaaliviivakoodi[VirtuaaliviivakoodiSlice.SYMBOL]
```